jekyll/_cci2/circleci-images.md

```diff
@@ -51,34 +52,34 @@ If you need a generic image to run on CircleCI, to use with orbs, or to use as a
 
 **Resources**
 
-You can find this image on [Docker Hub](https://hub.docker.com/r/cimg/base), and the source code and documentation on [GitHub](https://github.com/CircleCI-Public/cimg-base).
+You can find more config examples for this image on the [Developer Hub](https://circleci.com/developer/images/image/cimg/base), and the source code and documentation on [GitHub](https://github.com/CircleCI-Public/cimg-base).
 
-## Next-gen CircleCI Go Image
+## Next-gen CircleCI Images
+
+CircleCI is moving to a new image repository that brings better documentation and more determinism. Below is an example image definition for the next-gen Go image.
 
 ```yaml
 image: cimg/go:1.13
 ```
 
-This is a direct replacement for the legacy CircleCI Go image (`circleci/golang`). It brings with it better documentation, more determinism, and benefits from the highly efficient infrastructure it’s built on.
+This is a direct replacement for the legacy CircleCI Go image (`circleci/golang`). Note, the Docker Hub repository is `cimg`.
 
 
 ## Best Practices
 
-The convenience images in the following sections are based on the most recently built versions of upstream images, so it is best practice to use the most specific image possible. This makes your builds more deterministic by preventing an upstream image from introducing unintended changes to your image.
-
-CircleCI bases pre-built images off of upstream, for example, `circleci/ruby:2.4-node` is based off the most up to date version of the Ruby 2.4-node container. Using `circleci/ruby:2.4-node` is similar to using `:latest`. It is best practice to lock down aspects of your build container by specifying an additional tag to pin down the image in your configuration.
+The next-gen convenience images in the following sections are based on the most recent LTS Ubuntu docker images and installed with the base libraries for the language or services, so it is best practice to use the most specific image possible. This makes your builds more deterministic by preventing an upstream image from introducing unintended changes to your image. The legacy images are Debian `jessie` or `stretch`.
```

Comment:
> LTS Ubuntu docker

Should be: "Ubuntu LTS Docker"

Reply:
> The legacy images are Debian `jessie` or `stretch`

This is mostly true. Depending on the image, there's various other bases as well.

Reply:
I can just remove it. I just didn't want to not mention it since some of that content was dropping.